cmd/mirror-main.go

```diff
@@ -337,13 +337,17 @@ func (mj *mirrorJob) watchMirror(ctx context.Context, cancelMirror context.Cance
 					sourceURLFull = tmpSrcURL
 				}
 			}
+			eventPath := event.Path
 
-			sourceURL := newClientURL(event.Path)
-			aliasedPath := strings.Replace(event.Path, sourceURLFull, mj.sourceURL, -1)
+			// Strip the prefixes in the event path. Happens in darwin OS only
+			eventPath = eventPath[strings.Index(eventPath, sourceURLFull):len(eventPath)]
```

Comment:
If this happens only on darwin, we can limit the code to darwin only i.e perhaps bring an `if runtime.GOOS == darwin` check so that we do not disturb the existing code on all other platforms.

Reply:
@harshavardhana , The change wont disturb the other flows . will generate and check the other binaries as well !

Reply:
Updated the change